Source/CarthageKit/Errors.swift

```diff
@@ -42,6 +42,9 @@ public enum CarthageError: Error {
 	/// Failed to write a file or directory at the given URL.
 	case writeFailed(URL, NSError?)
 
+	/// No available simulators could be found
+	case noAvailableSimulators
```

Comment:
This should be tied to a SDK and output its platform name in a error message (e.g. `Could not find any available watchOS simulators`).

Reply:
Thank you @ikesyo 

I added `platformName` to `availableSimulators` error.